File: src/Frontend/projects/core/src/lib/models/dto/seasons/brand-policy-data.ts
export class BrandPolicyData {
    public id: number;
    public discountPolicyId: number;
    public createTime: Date;
    public brandName: string;
    public volume: number;
    public prepaymentVolumePercent: number;
    public cutoffDiscountPercent: number;

    static fromJSON(obj: any) {
        if (!obj) {
            return null;
        }
        return Object.assign(
            new BrandPolicyData(),
            obj,
            {
                createTime: obj.createTime ? new Date(obj.createTime) : null,
            },
        );
    }
}

File: src/Backend/Common/Models/Seasons/BrandPolicyData.cs
using System;

namespace Company.Common.Models.Seasons
{
    public class BrandPolicyData
    {
        public int Id { get; set; }

        public int DiscountPolicyId { get; set; }

        public DiscountPolicy DiscountPolicy { get; set; }

        public DateTime CreateTime { get; set; }

        public DateTime? DeleteTime { get; set; }

        public int CreatorId { get; set; }

        public int? DeleterId { get; set; }

        public string BrandName { get; set; }

        /// <summary>
        /// у.е.
        /// </summary>
        public float Volume { get; set; }

        /// <summary>
        /// %
        /// </summary>
        public float PrepaymentVolumePercent { get; set; }

        /// <summary>
        /// %
        /// </summary>
        public float CutoffDiscountPercent { get; set; }
    }
}
